src/org/joml/Vector2d.java

```diff
@@ -42,11 +42,27 @@ of this software and associated documentation files (the "Software"), to deal
  *
  * @author RGreenlees
  * @author Kai Burjack
+ * @author F. Neurath
  */
 public class Vector2d implements Externalizable, Vector2dc {
 
     private static final long serialVersionUID = 1L;
 
+    /**
+     * Gets the length of a 2-dimensional double-precision vector.
```

Comment:
Please change this to use imperative mood "Get" instead of second person present tense "Gets", like everywhere else.